holidays/countries/south_georgia_and_the_south_sandwich_islands.py

```diff
@@ -0,0 +1,160 @@
+        * [2020](https://web.archive.org/web/20250726192724/https://laws.gov.gs/wp-content/uploads/2022/05/2019.pdf)
+        * [2021](https://web.archive.org/web/20210911002738/https://www.gov.gs/docsarchive/Legislation/SGSSI%20Gazette%20No%202%20dated%2023%20December%202020.pdf)
+        * [2022](https://web.archive.org/web/20250727082227/https://laws.gov.gs/wp-content/uploads/2022/05/2021.pdf)
+        * [2023](https://web.archive.org/web/20240810165140/https://laws.gov.gs/wp-content/uploads/2024/03/Gazette2023.pdf)
+        * [2024](https://web.archive.org/web/20240810163638/https://laws.gov.gs/wp-content/uploads/2024/02/SGSSI-Gazette-No-1-dated-27-February-2024.pdf)
+        * [2025](https://web.archive.org/web/20250404025158/https://laws.gov.gs/wp-content/uploads/2025/01/SGSSI-Gazette-No-1-dated-30-January-2025.pdf)
+    """
+
+    country = "GS"
+    default_language = "en_GS"
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    supported_languages = ("en_GS", "en_US")
+    start_year = 2012
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, SouthGeorgiaAndTheSouthSandwichIslandsStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("New Year's Day")))
+
+        # Possession Day.
+        self._add_observed(self._add_holiday_jan_17(tr("Possession Day")))
+
+        # Good Friday.
+        self._add_good_friday(tr("Good Friday"))
+
+        if self._year <= 2019:
+            # Easter Monday.
+            self._add_easter_monday(tr("Easter Monday"))
+
+        if 2018 <= self._year <= 2022:
+            # The Queen's Birthday.
+            dt = self._add_holiday_apr_21(tr("The Queen's Birthday"))
+            if self._year not in {2019, 2020}:
+                self._add_observed(dt)
+
+        # Liberation Day.
+        name = tr("Liberation Day")
+        self._add_observed(
+            self._add_holiday_apr_25(name)
+            if self._year >= 2016
+            else self._add_holiday_apr_26(name)
+        )
+
+        if self._year >= 2015:
+            # Shackleton Day.
+            dt = self._add_holiday_may_20(tr("Shackleton Day"))
+            if self._year != 2023:
+                self._add_observed(dt)
+
+        # Mid-winter Day.
+        dt = self._add_holiday_jun_21(tr("Mid-winter Day"))
+        if self._year != 2025:
+            self._add_observed(dt)
+
+        if self._year <= 2021:
+            if self._year >= 2015:
+                # Toothfish Day.
+                self._add_observed(self._add_holiday_sep_4(tr("Toothfish Day")))
+            else:
+                # Toothfish (end of season) Day.
```

Comment:
```suggestion
                # Toothfish (End of Season) Day.
```
If it's in en_US locale.